extensions/wikia/AdEngine/js/PageFairDetection.js

```diff
@@ -0,0 +1,73 @@
+define('ext.wikia.adEngine.pageFairDetection', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.utils.scriptLoader',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (adContext, scriptLoader, doc, log, window) {
+	'use strict';
+
+	var context = null,
```

Comment:
won't `context = context || getContext()` work? This way you don't need a function.